index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+        <html lang="en">
+            <head>
+        <title>Nawaf Ibrhaim | portfolio</title>
+        <h1></h1>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    
+            </head>
+
+                <body>
+        
+            <nav><h2>
+                    <a href="file:///Users/icode/code/wdi/homework/hw-w02d01-html-and-css/index.html" >About Me </a>|
```

Comment:
this link will only work on your computer you need to add relative path not an absolute